src/routes/workshop-a11y-fundamentals.svelte

```diff
@@ -203,174 +550,348 @@
-      The only thing that I would change it's the duration of the workshop, maybe more time to try
-      to do everything
-    </q>
-    <q>
-      The quiz, in the end, was a nice and fun way to wrap up! Overall, really enjoying it, thanks!
-    </q>
-    <q>
-      It was one of the most brilliant introductions to a11y from a FE perspective, and the pace was
-      balanced.
-    </q>
-    <q>
-      The content and the pace were good, also your knowledge on this topic is really remarkable, it
-      was really an enjoyable moment.
-    </q>
-    <q>
-      Honestly, I really liked it. The only thing that I would change it's the duration of the
-      workshop, maybe more time to try to do everything
-    </q>
-    <q>
-      You rock sandrina, it was a pleasure ! We leave with the will to speak with you for hours
-      about development.
-    </q>
-    <q>
-      I enjoyed the workshop overall, you are a cool trainer, very accessible and very knowledgeable
-      on the matter.
-    </q>
+  <!-- <article class="t-card t-cta">
+    <h2 class="t-ctaTitle">Join the online workshop!</h2>
+    <p class="t-ctaPitch">Get 9 hours, over 3 days, to refine your A11Y knowledge.</p>
+
+    <dl class="t-ctaPoints">
+      <dt>
+        <svg aria-hidden="true" class="u-svg" style="display: none;">
+          <use xlink:href="#calendar" />
+        </svg>
+        <span class="sr-only">When:</span>
+      </dt>
+      <dd>To be defined (April?)</dd>
+
+      <dt>
+        <svg aria-hidden="true" class="u-svg" style="display: none;">
+          <use xlink:href="#clock" />
+        </svg>
+        <span class="sr-only">Hour:</span>
+      </dt>
+      <dd>{eventHour} <span class="tz">{eventTZ}</span></dd>
+    </dl>
+
+    <a href="" class="t-ctaBtn">Buy ticket for {price}</a>
+    <span class="t-ctaNote">Price is Early Bird</span>
+
+  </article> -->
+  <WorkshopForm formEndpoint={endpointA11Y}>
+    <p class="t-ctaPitch">Get 9 hours, over 3 days, to refine your A11Y knowledge.</p>
+
+    <dl class="t-ctaPoints">
+      <dt class="time-todo">
+        <svg aria-hidden="true" class="u-svg" style="display: none;">
+          <use xlink:href="#calendar" />
+        </svg>
+        <span class="sr-only">When:</span>
+      </dt>
+      <dd>Soon</dd>
+
+      <dt>
+        <svg aria-hidden="true" class="u-svg" style="display: none;">
+          <use xlink:href="#clock" />
+        </svg>
+        <span class="sr-only">Hour:</span>
+      </dt>
+      <dd>{eventHour} <span class="tz">{eventTZ}</span></dd>
+    </dl>
+  </WorkshopForm>
+
+
+
+  <h2 class="t-title asH2">
+    F.A.Q.
+  </h2>
+
+  <div class="faqArea">
+    <ul class="faqList">
+    <li>
+        <Accordion summary="What does A11Y mean?">
+          A11Y stands for 'accessibility'.
+          The irony about the word "accessibility" itself is that it's not very easy to pronounce.
+          So, it was created this abbreviation: A + 11 characters + Y, which has a more friendly homophone of "ally".
```

Comment:
The abbreviation stands for A + 11 characters + Y, which is a more user-friendly homophone of "ally".